scripts/smoke-test-dist.ps1

```
@@ -0,0 +1,80 @@
+#!/usr/bin/env pwsh
+param (
+  [Parameter(Mandatory = $true)]
+  [string]$Archive
+)
+
+$ErrorActionPreference = "Stop"
+
+function Stop-SmokeTest([string]$Message) {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Stop-SmokeTest' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)